src/txFactory/identityManagerTransactionFactory.ts

```diff
@@ -57,6 +57,26 @@ export function createAlastriaIdentity(web3, publicKey) {
   return transaction
 }
 
+/**
+
+ * @param web3
+ * @param publicKeyHash
+ */
+export function createAlastriaIdentityHash(web3, publicKeyHash) {
+  const transaction = Object.assign({}, config.basicTransaction)
+  transaction.gasLimit = 600000
+  const publicKeyCallData = web3.eth.abi.encodeFunctionCall(
+    config.contractsAbi.AlastriaPublicKeyRegistry.addKey,
```

Comment:
Here dani the function addKey has been renamed to addPublicKey, so it is the one to be used
```suggestion
    config.contractsAbi.AlastriaPublicKeyRegistry.addPublicKey,
```